installJenkinsInVm.sh

```diff
@@ -0,0 +1,10 @@
+
+#If a VM named mos-vm exists then start it up
+
+if (multipass info mos-vm)
+then
+  echo "Launching mos-vm"
+  multipass shell mos-vm
```

Comment:
If you do this in the script in an IF statement, you are going to be stuck in there. The shell or ssh command should be run after the script that builds the VM. For example:

`bash create-vm.sh && multipass shell vmname`